docker-compose.yml

```
@@ -0,0 +1,14 @@
+services:
+  todoapp-dev:
+    env_file: .env
+    build:
+      context: .
+      target: development
+      dockerfile: Dockerfile
+    ports: 
+     - 8000:5000
+    volumes:
+      - type: bind
+        source: ./todo_app
+        target: /app/todo_app
+    platform: linux/amd64
```

Not important but someone recently introduced me to it and I'll be leaning on heavily in future; adding a `pull_policy: build` argument to this will ensure that every `docker compose up` command always tries to rebuild the image. This may not be a good thing for you, but I've fallen into the trap a few times of forgetting to add a `--build` flag after I've made a change that isn't captured by the bind mount (e.g. to the Dockerfile itself) and then spent a while wondering why my changes wasn't doing anything!
```suggestion
    platform: linux/amd64
    pull_policy: build
```